CHANGELOG

```diff
@@ -1,5 +1,16 @@
 Changelog for ssm
 =================
+* Thu Jun 29 2023 Adrian Coveney <adrian.coveney@stfc.ac.uk> - 3.3.0-1
+ - Added destination queue to the log during startup to aid troubleshooting.
+ - Added check that the config file exists to allow for better error messages.
+ - Changed dependencies to limit python-ldap below 3.4.0 and python-daemon below 2.3.0.
+ - Changed rpmbuild config to use less OS-specific dependencies.
+ - Fixed read timeouts leading to a crash.
+ - Fixed command line to allow a relative file path for the DNs file.
```

Comment:
could this line be worded better? the command line itself hasn't been fixed?

Reply:
How about, "Fixed command line arguments to..." ?

Reply:
Yeah 👍

Reply:
```suggestion
 - Fixed command line arguments to allow a relative file path for the DNs file.
```